test/controllers/IllnessEndDateController.test.ts

```diff
@@ -143,14 +160,14 @@ describe('IllnessEndDateController', () => {
         it('should return 422 response with rendered error messages when empty end date was submitted', async () => {
             const app = createApp({appeal});
             await request(app).post(ILLNESS_END_DATE_PAGE_URI)
-                .send({})
+                .send({ illnessStart: '2020-05-01' })
```

Comment:
We do not post illnessStart, better remove it, and in the other post, please. Thanks mate :)